lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -117,6 +117,24 @@ namespace Js
         return NewWithBufferT<LiteralString, true>(content, cchUseLength, scriptContext);
     }
 
+    JavascriptString* JavascriptString::NewWithByteCodeBuffer(__in_ecount(cchUseLength) const char16* content, charcount_t cchUseLength, Utf8SourceInfo* utf8SourceInfo, ScriptContext* scriptContext)
```

Comment:
Who calls this?

Reply:
oh it's in chakrafull nvm

---
In reply to: [198562798](https://github.com/Microsoft/ChakraCore/pull/5374#discussion_r198562798) [](ancestors = 198562798)